src/tools/cursors/index.js

```diff
@@ -0,0 +1,381 @@
+import MouseCursor from './MouseCursor.js';
+
+export default {
+  angleCursor,
+  arrowAnnotateCursor,
+  bidirectionalCursor,
+  cobbAngleCursor,
+  ellipticalRoiCursor,
+  freehandMouseCursor,
+  freehandSculpterMouseCursor,
+  lengthCursor,
+  probeCursor,
+  rectangleRoiCursor,
+  textMarkerCursor,
+  crosshairsCursor,
+  eraserCursor,
+  magnifyCursor,
+  panCursor,
+  rotateCursor,
+  stackScrollCursor,
+  wwwcRegionCursor,
+  wwwcCursor,
+  zoomCursor,
+};
```

Comment:
I don't think the exported constants are hoisted above the default export. Because this is what we're tying to `lib`'s export, these come back as `undefined` for me:

![image](https://user-images.githubusercontent.com/5797588/54793886-0b5d4280-4c1b-11e9-9f4a-4b5f7b2fc620.png)

Reply:
Aha, good catch. Thanks for fixing it.